src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -471,18 +475,26 @@ public Task SortByColumnAsync(ColumnBase<TGridItem> column, SortDirection direct
     /// </summary>
     /// <param name="title">The title of the column to sort by.</param>
     /// <param name="direction">The direction of sorting. The default is <see cref="SortDirection.Auto"/>. If the value is <see cref="SortDirection.Auto"/>, then it will toggle the direction on each call.</param>
+    /// <returns>A <see cref="Task"/> representing the completion of the operation.</returns>
     public Task SortByColumnAsync(string title, SortDirection direction = SortDirection.Auto)
     {
         var column = _columns.FirstOrDefault(c => c.Title?.Equals(title, StringComparison.InvariantCultureIgnoreCase) ?? false);
 
         return column is not null ? SortByColumnAsync(column, direction) : Task.CompletedTask;
     }
 
+    /// <summary>
+    /// Removes the grid's sort on double click for the currently sorted column if it's not a default sort column.
+    /// </summary>
+    /// <returns>A <see cref="Task"/> representing the completion of the operation.</returns>
+    public Task RemoveSortByColumnAsync() => (_sortByColumn != null) ? RemoveSortByColumnAsync(_sortByColumn) : Task.CompletedTask;
+
```

Comment:
Although a nice addition, this is not relevant for the PR subject. Please remove and create a new PR for this enhancement

Reply:
Sure, I'll move this change into another PR as you sugested. (I will only be able to do this next Monday)